temporalcli/commandsmd/commands.yml

```diff
@@ -0,0 +1,2890 @@
+      ```
+    option-sets:
+      - client
+  - name: temporal workflow cancel
+    summary: Send cancellation to Workflow Execution
+    description: |
+      Canceling a running Workflow Execution records a
+      `WorkflowExecutionCancelRequested` event in the Event History. The Service
+      schedules a new Command Task, and the Workflow Execution performs any cleanup
+      work supported by its implementation.
+
+      Use the Workflow ID to cancel an Execution:
+
+      ```
+      temporal workflow cancel \
+          --workflow-id YourWorkflowId
+      ```
+
+      A visibility Query lets you send bulk cancellations to Workflow Executions
+      matching the results:
+
+      ```
+      temporal workflow cancel \
+          --query YourQuery
+      ```
+
+      Visit https://docs.temporal.io/visibility to read more about Search Attributes
+      and Query creation. See `temporal batch --help` for a quick reference.
+    option-sets:
+      - single-workflow-or-batch
+  - name: temporal workflow count
+    summary: Number of Workflow Executions
+    description: |
+      Show a count of Workflow Executions, regardless of execution state (running,
+      terminated, etc). Use `--query` to select a subset of Workflow Executions:
+
+      ```
+      temporal workflow count \
+          --query YourQuery
+      ```
+
+      Visit https://docs.temporal.io/visibility to read more about Search Attributes
+      and Query creation. See `temporal batch --help` for a quick reference.
+    options:
+      - name: query
+        type: string
+        short: q
+        description: Content for an SQL-like `QUERY` List Filter.
+  - name: temporal workflow delete
+    summary: Remove Workflow Execution
+    description: |
+      Delete a Workflow Executions and its Event History:
+
+      ```
+      temporal workflow delete \
+          --workflow-id YourWorkflowId
+      ```
+
+      The removal executes asynchronously. If the Execution is Running, the Service
+      terminates it before deletion.
+
+      Visit https://docs.temporal.io/visibility to read more about Search Attributes
+      and Query creation. See `temporal batch --help` for a quick reference.
+    option-sets:
+      - single-workflow-or-batch
+  - name: temporal workflow describe
+    summary: Show Workflow Execution info
+    description: |
+      Display information about a specific Workflow Execution:
+
+      ```
+      temporal workflow describe \
+          --workflow-id YourWorkflowId
+      ```
+
+      Show the Workflow Execution's auto-reset points:
+
+      ```
+      temporal workflow describe \
+          --workflow-id YourWorkflowId \
+          --reset-points true
+      ```
+    option-sets:
+      - workflow-reference
+    options:
+      - name: reset-points
+        type: bool
+        description: Show auto-reset points only.
+      - name: raw
+        type: bool
+        description: Print properties without changing their format.
+        default: true
```

Comment:
Description for this option used to be 
```
* `--raw` (bool) -
  Print properties without changing their format.
  Defaults to true.
```
Which states defaults to `true`, but the code didn't seem to support that. Should this be removed from the description? A bool that defaults to true feels weird/doesn't make sense. But is this API changable at this point?